app/models/adjustment.rb

```diff
@@ -30,6 +30,14 @@ def self.storage_locations_adjusted_for(organization)
     includes(:storage_location).where(organization_id: organization.id).collect(&:storage_location)
   end
 
+  def split_difference
```

Comment:
I love how simple you guys made this look!

Reply:
Not gonna lie, this shit is poetry.

this took some serious noodling - @mdworken had some good ideas for breaking through the block I had hit on sorting this out.